// angular-star-rating/index.js
var __decorate = (this && this.__decorate) || function (decorators, target, key, desc) {
    var c = arguments.length, r = c < 3 ? target : desc === null ? desc = Object.getOwnPropertyDescriptor(target, key) : desc, d;
    if (typeof Reflect === "object" && typeof Reflect.decorate === "function") r = Reflect.decorate(decorators, target, key, desc);
    else for (var i = decorators.length - 1; i >= 0; i--) if (d = decorators[i]) r = (c < 3 ? d(r) : c > 3 ? d(target, key, r) : d(target, key)) || r;
    return c > 3 && r && Object.defineProperty(target, key, r), r;
};
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { StarRatingComponent } from './src/star-rating.component';
import { StarRatingConfig } from './src/star-rating-config';
export * from './src/star-rating.component';
export * from './src/star-rating-config';
var StarRatingModule = StarRatingModule_1 = (function () {
    function StarRatingModule() {
    }
    StarRatingModule.forRoot = function () {
        return {
            ngModule: StarRatingModule_1,
            providers: [StarRatingConfig]
        };
    };
    return StarRatingModule;
}());
StarRatingModule = StarRatingModule_1 = __decorate([
    NgModule({
        imports: [
            CommonModule
        ],
        declarations: [
            StarRatingComponent
        ],
        exports: [
            StarRatingComponent
        ]
    })
], StarRatingModule);
export { StarRatingModule };
var StarRatingModule_1;
